lib/manager/components/ProjectSettings.js

```diff
@@ -25,66 +19,33 @@ type Props = {
 }
 
 export default class ProjectSettings extends Component<Props> {
-  messages = getComponentMessages('ProjectSettings')
-
   _updateProjectSettings = (project: Project, settings: Object) => {
-    const {updateProject} = this.props
+    const { updateProject } = this.props
     // Update project and re-fetch feeds.
     updateProject(project.id, settings, true)
   }
 
   render () {
     const {
-      activeSettingsPanel,
```

Comment:
Remove this prop from type `Props`, and remove the corresponding passed prop from `ProjectViewer` line 268.